xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -2651,7 +2651,15 @@ void CActiveAE::LoadSettings()
   m_settings.resampleQuality = static_cast<AEQuality>(settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_PROCESSQUALITY));
   m_settings.atempoThreshold = settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_ATEMPOTHRESHOLD) / 100.0;
   m_settings.streamNoise = settings->GetBool(CSettings::SETTING_AUDIOOUTPUT_STREAMNOISE);
-  m_settings.silenceTimeout = settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_STREAMSILENCE) * 60000;
```

Comment:
Like I alluded to in the last PR.

Why not rename `silenceTimeout` to `silenceTimeoutMinutes` and not do any conversion here and on the receiving side use std::chrono::minutes` that way no other conversion between units is needed.

Reply:
Because is a fix, not a improvement with backport in mind (and keep it simple).

Reply:
This is what I had in mind: https://github.com/lrusak/xbmc/commit/f81b385370bdb158f8a6c5d4c4ce6c98394efca2

Reply:
Also looks good. Please sync internally among you guys. Feel free to superseed this one / change it / use it - the way you guys want it.

Reply:
My opinion regarding this will not change but since it is no longer my PR I have to accept what the OP prefers...